google_shopping.js

```diff
@@ -1,5 +1,59 @@
 var data = require("./products.json")
+var items = data.items;
+var counter = 0;
+var images = [];
+var canon = [];
+var eBayCanon = [];
 
 // Write your solutions below
 
-console.log(data["items"]);
\ No newline at end of file
+// console.log(data["items"]);
+
+for( var i = 0; i < items.length; i++){
+	if(items[i].kind == "shopping#product"){
+		counter++;
+		console.log('found product');
+	}
+}
+console.log(counter);
+
+for(var i =0; i < items.length; i++){
+	if(items[i].product.inventories[0].availability == "backorder"){
+		console.log(items[i].product);
+	}
+}
+
+
+for( var i = 0; i < items.length; i++){
+	var brand = items[i].product.brand;
+	var price = items[i].product.inventories[0].price;
+	var imageLink = items[i].product.images[0].link
+}
+
+for( var i = 0; i < items.length; i++){
+	if(items[i].product.images.length > 1){
+		images+= items[i].product.title + "\n";
+	console.log(images);
+	}
+}
+for( var i = 0; i < items.length; i++){
```

Comment:
Similar to above, let's move the console log outside of the loop and use .push() to append new products to the array. We can use .toLowerCase() to ignore case when searching for the brand.

```if(items[i].product.brand.toLowerCase() === "canon"){```
		```canon.push( items[i].product.title );```
	```}```
```}```
console.log(canon);